src/server/game/Grids/Notifiers/GridNotifiers.h

```diff
@@ -1296,6 +1319,33 @@ namespace Trinity
             NearestCreatureEntryWithLiveStateInObjectRangeCheck(NearestCreatureEntryWithLiveStateInObjectRangeCheck const&) = delete;
     };
 
+    class NearestCreatureEntryNotInListCheck
+    {
+    public:
+        NearestCreatureEntryNotInListCheck(WorldObject const& obj, uint32 entry, bool alive, float range, std::list<ObjectGuid> creatureGuids)
+            : i_obj(obj), i_entry(entry), i_alive(alive), i_range(range), i_creatureGuids(creatureGuids) { }
```

Comment:
```cpp
/home/circleci/project/src/server/game/Grids/Notifiers/GridNotifiers.h:1326:59:
 fatal error: field 'i_range' will be initialized after field 'i_creatureGuids' [-Wreorder]
            : i_obj(obj), i_entry(entry), i_alive(alive), i_range(range), i_creatureGuids(creatureGuids) { }
                                                          ^
```
You should rearrange the last 2 arguments: `, i_creatureGuids(creatureGuids, i_range(range)) { }`